docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[release-notes-header-8.6.0]]
+== 8.6
+
+[discrete]
+[[release-notes-8.6.0]]
+=== 8.6.0
+
+[discrete]
+[[known-issue-8.6.0]]
+==== Known issues
+There are no known issues in 8.6.0.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+==== Breaking changes
+There are no breaking changes in 8.6.0.
+
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+// end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.6.0]]
+==== Deprecations
+There are no deprecations in 8.6.0.
+
+
+[discrete]
+[[features-8.6.0]]
+==== Features
+* Allows you to add indicators to new or existing cases ({pull}145121[#145121]).
+* Adds the `is one of` operator to the *Add field* menu in Timeline ({pull}144988[#144988]).
+* Adds an "Add to timeline investigation" button to the User Risk Scores and Host Risk Scores cards on the Entity Analytics dashboard ({pull}144819[#144819]).
+* Provides the option to duplicate rules and their exceptions or rules only ({pull}144782[#144782]).
+* Improves the Rule Exceptions interface and allows you to export read-only exception lists ({pull}144383[#144383]).
+* Enables a technical preview of functionality that allows users to create runtime queries that can be parameterized from alert data or hard coded literal values. ({pull}145240[#145240]).
+* Allows users to deploy Osquery across all {agent} policies or on specified policies only ({pull}143948[#143948]).
```

Comment:
```suggestion

```

Reply:
Note to self that I'll likely need to add this to the Kibana release notes since this feature was developed for the Kibana Osquery page.

Reply:
Added to Kibana release notes in https://github.com/elastic/kibana/pull/148480.